packages/test/test-end-to-end-tests/src/test/blobs.spec.ts

```diff
@@ -449,7 +449,7 @@ describeNoCompat("Garbage collection of blobs", (getTestObjectProvider) => {
         const gcContainerConfig: ITestContainerConfig = {
             runtimeOptions: {
                 gcOptions: {
-                    gcAllowed: true, runGCInTestMode: deleteUnreferencedContent, writeDataAtRoot: true,
+                    gcAllowed: true, runGCInTestMode: deleteUnreferencedContent,
```

Comment:
In the `mockConfigProivder.ts` we have `settings["Fluid.GarbageCollection.WriteDataAtRoot"] = "true";`. Delete that as well?

Reply:
Removed all usages except 1 in `gcUnknownHandles.spec.ts` where its needed for back-compat - Older container runtime doesn't have this enabled by default.